db-client-java/src/main/java/com/eventstore/dbclient/GossipClient.java

```diff
@@ -25,7 +26,7 @@ public GossipClient(ManagedChannel channel) {
     }
 
     public void shutdown() throws InterruptedException {
-        _channel.shutdown().awaitTermination(Timeouts.DEFAULT.shutdownTimeout, Timeouts.DEFAULT.shutdownTimeoutUnit);
+        _channel.shutdown().awaitTermination(5, TimeUnit.SECONDS);
```

Comment:
Shouldn't it be 10 seconds? https://github.com/EventStore/architecture-and-planning/blob/main/rfcs/032%20-%20grpc-clients-deadlines.md#default-timeouts

Reply:
I would argue that deadlines are for operations only and nothing in the RFC mentions anything for connection management. The Java gRPC lib is the only one with an async close function. I set five because it was the previous default value for shutdowns.